src/ppx/geometry.cpp

```diff
@@ -19,6 +19,334 @@
 
 namespace ppx {
 
+// -------------------------------------------------------------------------------------------------
+// VertexDataProcessor
+// -------------------------------------------------------------------------------------------------
+class IVertexDataProcessor
```

Comment:
Are you adding "I" because it is an abstract class? I don't think any other classes follow this naming convention in BigWheels

Reply:
changed the class name to VertexDataProcessorBase